redbot/cogs/audio/converters.py

```diff
@@ -376,17 +376,13 @@ class ComplexScopeParser(commands.Converter):
                 to_user_raw = str(to_user_raw)
             if target_user is None:
                 member_converter = commands.MemberConverter()
-                user_converter = commands.UserConverter()
```

Comment:
Is there a reason why you decided to remove the user converter here? this is supposed to be able to interact with users both inside and outside a given server

Reply:
And on line 421

Reply:
Yes, the user converter calls fetch_user

Reply:
ohhh since when :eyes:  fuck ... okay yeah i'll get the ID pr done today as this need to be replaced lmao

Reply:
Huh, no, apparently it doesnt. I remember there being discussion about this, but....